tests/support/utils.ts

```diff
@@ -0,0 +1,33 @@
+import bs58 from "bs58";
```

Comment:
Why are we copying these functions here? Can't we just use the ones in `lib/utils`?

Reply:
I copied the functions, since I think we should make use of them instead of using `substring` everywhere.
But I wasn't able to use `lib/utils` directly since we import `config.json` in `config.ts` which requires type assertions from node
https://v8.dev/features/import-assertions
Which means we need to target a higher module in our typescript config like `esnext`

So either we go back to use `substring` or maybe update the typescript module?

Reply:
I'm fine with using either `substring` or just hardcoding the expected results as strings in tests.